packages/amplify_datastore/example/lib/main.dart

```diff
@@ -177,6 +178,15 @@ class _MyAppState extends State<MyApp> {
     });
   }
 
+  getPost() async {
```

Comment:
Leaving the return type unspecified makes it implictly `dynamic`. The other code in this file sets a bad example. Also, is this supposed to be singular? This prints all the posts currently.